### PR TITLE
Fix error when trying to find files that changed on master

### PR DESCRIPTION
**Pull request by zephraph**
See original failure here: https://circleci.com/gh/artsy/reaction/53222

When finally adding `lint-changed` to reaction, it failed with the following error

```
Unable to get changed files since last tag
```

Unfortunately I hadn't added very explicit error handling previously so it was hard to know what failed. After adding some error handling and doing more digging I found this error:

image

This was just a raw test, so the diff isn't exactly the same (it would be `v3.0.0..HEAD` in the real case), but the important thing to note here is the newline. Because I'm running git as a subprocess and grabbing stdout, it's appending a newline in some cases. I'd fixed it in some of the other commands, but unfortunately this case still stuck around. 

To be more robust, I put a `trim` statement in the top level git command. After further testing, everything works well. 👍 

<details>
  <summary>📦 Published PR as canary version: <code>3.0.1-canary.3.67.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```sh
  npm install @artsy/lint-changed@3.0.1-canary.3.67.0
  # or 
  yarn add @artsy/lint-changed@3.0.1-canary.3.67.0
  ```
</details>

### CODE REVIEW

File: src/lint-changed.ts [hunk 1]
```diff
@@ -35,12 +35,12 @@ async function runCommand(cmdString: string) {
   if (stderr) {
     throw new Error(stderr);
   }
-  return stdout;
+  return stdout.trim();
```
Comment (zephraph) [inline src/lint-changed.ts]: This was ultimately the issue. The `gitLastTag` method returned the tag name padded with a newline. That actually broke the diffing mechanism to find file changes.

**Comment by zephraph** [comment]
cc @damassi, @eessex

File: src/lint-changed.ts (hunk 1 above)
Comment (damassi) [reply src/lint-changed.ts]: 😄

**Comment by artsyit** [comment]
:rocket: PR was released in `v3.0.1` :rocket:

**Pull request merged**